### PR TITLE
[HOTFIX][CH] Fix randomly failed ut for PR#3537

### PR DESCRIPTION
**Pull request by zzcclp**
## What changes were proposed in this pull request?

There are some randomly failed ( result not match ) when running ut for PR #3537.

(Fixes: \#ISSUE-ID)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Review by liuneng1994 — APPROVED** [review]

File: backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala [hunk 1]
```diff
@@ -2161,53 +2161,6 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
     }
   }
 
-  test("GLUTEN-3534: Fix incorrect logic of judging whether supports pre-project for the shuffle") {
```
Comment (baibaichen) [inline backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala:2164]: Let's ignore it first. I think it more like a bug, we must to know the RC.

Comment (zzcclp) [reply backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala:2164]: already created the #3731 to trace this issue.

Comment (baibaichen) [reply backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala:2164]: OK, We can revert removed test easily, let's go on.

**Review by baibaichen — DISMISSED** [review]

**Review by baibaichen — APPROVED** [review]
LGTM

**Pull request merged**